File: Client/src/app/models/GetAllCookedRecipesVM.ts
import { CookedRecipeDTO } from "./CookedRecipeDTO";
import { RecipesOptionDTO } from "./RecipesOptionDTO";
import { UnitTypeDTO } from "./UnitTypeDTO";

export class GetAllCookedRecipesVM {
  kitchenUnitTypes?: UnitTypeDTO[] = undefined;
  cookedRecipes?: CookedRecipeDTO[] = undefined;
  recipesOptions?: RecipesOptionDTO[] = undefined;
}

File: API/ContainerNinja.Contracts/ViewModels/GetAllCookedRecipesVM.cs
using ContainerNinja.Contracts.DTO;

namespace ContainerNinja.Contracts.ViewModels
{
    public record GetAllCookedRecipesVM
    {
        public List<CookedRecipeDTO> CookedRecipes { get; set; }
        public List<KitchenUnitTypeDTO> KitchenUnitTypes { get; set; }
        public List<RecipesOptionDTO> RecipesOptions { get; set; } = new List<RecipesOptionDTO>();
    }
}